File: Iniciante/URI 1011.py
import math

pi = None
raio = None
volume = None

def read_numeric():
  try:
    # read for Python 2.x
    return float(raw_input())
  except NameError:
    # read for Python 3.x
    return float(input())

pi = 3.14159
raio = read_numeric()
volume = ((4 * pi) * (math.pow(raio, 3))) / 3
print(str("VOLUME = ") + str("{:0.3f}".format(volume)))

File: Iniciante/URI 1011.pas
uses Math;

const
    pi = 3.14159;

var
	raio : double;
	volume : double;

begin
    read(raio);
	
    volume := (4.0 / 3) * pi * power(raio, 3.0);
	
    writeln('VOLUME = ', volume:0:3);
end.